src/proxy-ui-api/frontend/tests/e2e/pages/ssMainPage.js

```diff
@@ -693,374 +135,75 @@ module.exports = {
       commands: [serviceDetailsCommands],
-      elements: {
-        serviceDetailsCloseButton:
-          '//*[contains(@class, "cert-dialog-header")]//*[contains(@id, "close-x")]',
-        deleteServiceButton:
-          '//button[@data-test="service-description-details-delete-button"]',
-        confirmDeleteButton: '//button[@data-test="dialog-save-button"]',
-        cancelDeleteButton: '//button[@data-test="dialog-cancel-button"]',
-        serviceURL: '//*[contains(@class, "url-input")]//input',
-        serviceCode:
-          '//*[contains(@class, "code-input")]//input[@name="code_field"]',
-        serviceType:
-          '//div[@data-test="service-description-details-url-type-value"]',
-        URLMessage:
-          '//*[contains(@class, "validation-provider")]//*[contains(@class, "v-messages__message")]',
-        codeMessage:
-          '//*[contains(@class, "code-input")]//*[contains(@class, "v-messages__message")]',
-        confirmDialogButton:
-          '//button[@data-test="service-description-details-save-button"]',
-        cancelDialogButton:
-          '//button[@data-test="service-description-details-cancel-button"]',
-      },
+      elements: serviceDetailsElements,
     },
     wsdlOperationDetails: {
       selector:
         '//div[contains(@class, "base-full-width") and .//div[contains(@class, "apply-to-all-text")]]',
-      commands: [wsdlOperationCommands],
-      elements: {
-        parametersTab: '//*[@data-test="parameters"]',
-        endpointsTab: '//*[@data-test="endpoints"]',
-        urlApplyToAllCheckbox: '//input[@data-test="url-all"]/following-sibling::div',
-        timeoutApplyToAllCheckbox: '//input[@data-test="timeout-all"]/following-sibling::div',
-        verifyCertApplyToAllCheckbox: '//input[@data-test="ssl-auth-all"]/following-sibling::div',
-        serviceURL: '//input[@data-test="service-url"]',
-        timeout: '//input[@data-test="service-timeout"]',
-        sslAuth: '//input[@data-test="ssl-auth"]',
-        sslAuthClickarea: '//input[@data-test="ssl-auth"]/following-sibling::div',
-        save2Button: '//button[@data-test="save-service-parameters"]/following-sibling::div',
-        saveButton: '//button[@data-test="save-service-parameters"]',
-        closeButton: '//button[@data-test="close"]',
-        urlHelp: '//div[@data-test="service-parameters-service-url-label"]//i',
-        timeoutHelp: '//div[@data-test="service-parameters-timeout-label"]//i',
-        verifyCertHelp: '//div[@data-test="service-parameters-verify-tls-label"]//i',
-        activeTooltip: '//div[contains(@class, "v-tooltip__content") and contains(@class,"menuable__content__active")]//span',
-        addSubjectsButton: '//button[@data-test="show-add-subjects"]',
-        removeAllButton: '//button[@data-test="remove-subjects"]',
-      },
+      commands: [operationDetailsCommands],
```

Comment:
These arrays are not necessary as long as there is only one import inside of it - it's not an issue but i wonder whether we should establish some sort of common practice that array is always created regardless if it has only one member or other way around, never use array for single reference.

Reply:
I have seen these single-item array mandatory even thought not required by the documentation. If left out the test cases fail due to not finding those commands.